Formula/file-roller.rb

```diff
@@ -51,6 +69,6 @@ def post_install
   end
 
   test do
-    system "#{bin}/file-roller", "--version"
+    assert_match version.to_s, shell_output("#{bin}/file-roller --version 2>/dev/null")
```

Comment:
The thought there was that if anything of interest actually occurred, the exit code would be non zero, and then the investigation could ensue, but I'll take it out :)